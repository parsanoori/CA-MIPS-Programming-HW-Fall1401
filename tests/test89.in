91
562496549
868679064
26509700
470598493
88484587
631213426
516234850
368885346
667824531
43065958
660183878
69534875
915069991
165195237
445366825
664475423
643360689
788453688
350574292
235360229
351956743
791173511
602022124
204103905
412595234
471734223
307768819
307614013
735067764
62555069
834189902
8356049
771019415
594639917
687527864
389435359
893798910
6342815
16670057
54846036
699219563
335834884
324101586
450543556
621306045
863519483
1026326085
34036389
281118275
474814632
766820723
9705792
563190323
480397691
18299284
327513822
474179907
19064209
890664786
594723987
528878364
974331307
255631434
84765964
1049347290
1061837095
284984309
601992055
459703519
64108672
773033191
303576081
512566887
448068843
629542711
165354577
513517405
571721278
462395594
21127267
751674903
629053157
380101913
677615608
91134470
113974714
490115739
236316267
185245442
467389984
293373632