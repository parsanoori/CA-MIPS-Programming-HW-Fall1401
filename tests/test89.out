6342815
8356049
9705792
16670057
18299284
19064209
21127267
26509700
34036389
43065958
54846036
62555069
64108672
69534875
84765964
88484587
91134470
113974714
165195237
165354577
185245442
204103905
235360229
236316267
255631434
281118275
284984309
293373632
303576081
307614013
307768819
324101586
327513822
335834884
350574292
351956743
368885346
380101913
389435359
412595234
445366825
448068843
450543556
459703519
462395594
467389984
470598493
471734223
474179907
474814632
480397691
490115739
512566887
513517405
516234850
528878364
562496549
563190323
571721278
594639917
594723987
601992055
602022124
621306045
629053157
629542711
631213426
643360689
660183878
664475423
667824531
677615608
687527864
699219563
735067764
751674903
766820723
771019415
773033191
788453688
791173511
834189902
863519483
868679064
890664786
893798910
915069991
974331307
1026326085
1049347290
1061837095